lab-will/test/cat-route-test.js

```diff
@@ -36,4 +50,29 @@ describe('Cat Routes', function() {
       });
     });
   });
+
+  describe('GET /api/cat', function() {
+    it('should return not found', function(done) {
+      request.get(`localhost:8000/api/cat?id=${-50}`)
+      .end((err, res) => {
+        if (err) return done (err);
+        expect(res.status).to.equal(404);
+        // expect(res.body.name).to.equal('test name');
+        // expect(res.body.content).to.equal('test content');
+        done();
+      });
+    });
+  });
+
+  describe('GET /api/cat', function() {
+    it('should return not found', function(done) {
+      request.get(`localhost:8000/api/cat?`)
+      .end((err, res) => {
+        if (err) return done (err);
+        expect(res.status).to.equal(400);
+        done();
+      });
+    });
+  });
+
 });
```

Comment:
These tests look pretty good.  You should add a test for your delete route though.